src/less/carousel/base/carousel.less

```diff
@@ -13,6 +13,54 @@
     }
 }
 
+.carousel-control-mixin(@control-height; @control-width) {
```

Comment:
There is a new organization system for mixins.  All mixins are in the `mixins` directory.  Can you move all the mixins there?
Same thing is happening for the variables. There is a `variables` directory. This would probably be a good time to move those too.

Reply:
Also, do we need a mixin for all the control styles, even though only 2 variables are being used?  Couldn't we define these as variables like `carousel-control-height` and avoid the mixin?